### PR TITLE
[PACT-704] Added stub service for updating an Edit Set Record.

### PR DESCRIPTION
**Pull request by tna-erasmos**
Please note that the record isn't actually updated.

### CODE REVIEW

File: app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala [hunk 1]
```diff
@@ -64,11 +66,29 @@ class ResponseBuilder[F[_] : ME : Logger] {
     asGetEditSetRecordRequest(jmsMessage).flatMap(getEditSetRecordRequest =>
       getEditSetRecordByOCI(getEditSetRecordRequest.recordOci)
         .map(editSetRecord => me.pure(Json.toJson(editSetRecord).toString))
-        .getOrElse(
-          onUnhandledCase(
-            s"Unable to find record for Edit Set with OCI [${getEditSetRecordRequest.editSetOci}] and Record OCI [${getEditSetRecordRequest.recordOci}]"
-          )
+        getOrElse onUnknownEditSetRecord(getEditSetRecordRequest.editSetOci, getEditSetRecordRequest.recordOci)
```
Comment (felixgb) [inline app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala:69]: The method call syntax doesn't match the rest of the project here, elsewhere we use the `.` syntax.

Comment (tna-erasmos) [reply app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala:69]: Sorry, how do you mean?

Comment (tna-erasmos) [reply app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala:69]: Oh - I'm missing the dot before `getOrElse` - odd that it works. Will change.

Comment (tna-erasmos) [reply app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala]: Changed. I was surprised that it compiled.

**Review by rwalpole — APPROVED** [review]

**Pull request merged**